### ISSUE TITLE
errata in stream-flows-and-basics docs

### ISSUE THREAD

**Issue by ijsong** [akka/akka#17074]
There is an errata in akka stream document (http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0-M4/scala/stream-flows-and-basics.html).

Comment in example code of section "Defining sources, sinks and flows" has some errata: 
```
// (X) Create a source form a Future
// (O) Create a source from a Future
```

**Comment by 2m** [comment]
Fixed by #17075.

**Comment by 2m** [comment]
Not yet fixed, because fix was merged to a wrong branch.
